Firestore/Source/Public/FIRTimestamp.h

```diff
@@ -62,6 +62,13 @@ NS_SWIFT_NAME(Timestamp)
 /** Returns a new NSDate corresponding to this timestamp. This may lose precision. */
 - (NSDate *)dateValue;
 
+/**
+ * Returns the result of comparing the receiver with another timestamp.
+ * @param other the other timestamp to compare.
+ * @return .orderedAscending if `other` is chronologically following self,
```

Comment:
Question: these are Swift-specific names for these enumeration values--is that something we're doing across all docs in all SDKs?

The equivalent `NSOrderedAscending` gives plenty of guidance on how to migrate it to Swift, but searching for ".orderedAscending Swift" gives no guidance for how to use it in Objective-C. Given that Objective-C is still a first class platform this seems suboptimal.

Reply:
It's not. I deliberated between the two and plain English before arbitrarily choosing the Swift syntax.

Fixed by replacing with ObjC syntax.